src/main/java/com/alipay/oceanbase/rpc/ObClusterTableQuery.java

```diff
@@ -39,7 +39,13 @@ public class ObClusterTableQuery extends AbstractTableQuery {
     public ObClusterTableQuery(ObTableClientQueryImpl tableQuery) {
         this.tableClientQuery = tableQuery;
     }
-
+    
```

Comment:
In my opinion, "[Feat] aggregation impl" is a better commit for this merge.
You could commit "[Fix] review" in the next few commits.

Reply:
By the way, _resolved conversation_ should be closed by the reviewer.

Reply:
get